extensions/blocks/mailchimp/view.scss

```
@@ -12,6 +12,16 @@
 		margin-bottom: 1em;
 	}
 
+	input[name='email'] {
+		&.error:focus,
+		&.error {
```

Do we need the added specificity here? We only have one `.error` in the block and the styles are scoped to that block, right? We'd probably want to swap the order around to have the modifier second too.

```suggestion
	.error,
	.error:focus {
```

---

You got a point over there! Resolved